server/lib/pbench/__init__.py

```diff
@@ -145,7 +145,10 @@ def get_pbench_logger(caller, config):
 
     pbench_logger = logging.getLogger(caller)
     if caller not in _handlers:
-        pbench_logger.setLevel(logging.DEBUG)
+        if config.logging_level == 'DEBUG':
```

Comment:
Why not do:

     pbench_logger.setLevel(config.logging_level)

except that we probably need to map it to the level defined in the logging module, but the point is tthat we should not limit ourselves to just DEBUG or INFO: we should be able to set it to whatever the config file tells us.

Also the default level is INFO if it's not defined in the config fie - is that what we want?

Reply:
Yaa I did same. earlier it wasn't working but now it is.
and yes the issue regarding what to keep as default setting level still remains.